efficiency/src/helper.cc

```diff
@@ -267,130 +296,34 @@ std::vector<float> getPtBounds()
     return pt_boundaries;
 }
 
-//__________________________________________________________________________________________
-/*std::vector<int> matchedSimTrkIdxs(std::vector<int> hitidxs, std::vector<int> hittypes)
+std::vector<float> getPtBounds(int mode)
```

Comment:
this should replace `std::vector<float> getPtBounds()` with smth like `return getPtBounds(ana.ptbound_mode)` to keep consistent ranges/definitions

Reply:
This code wasn't written by me but I agree. It seems like the function without a mode input is never used so I'm just going to remove it.